src/deadline/client/cli/_groups/queue_group.py

```diff
@@ -48,6 +50,99 @@ def queue_list(**args):
     click.echo(_cli_object_repr(structured_queue_list))
 
 
+@cli_queue.command(name="export-credentials")
+@click.option("--queue-id", help="The queue ID to use.")
+@click.option("--farm-id", help="The farm ID to use.")
+@click.option(
+    "--mode",
+    type=click.Choice(["USER", "READ"], case_sensitive=False),
+    default="USER",
+    help="The type of queue role to assume (default: USER)",
+)
+@click.option(
+    "--output-format",
+    type=click.Choice(["credentials_process"], case_sensitive=False),
+    default="credentials_process",
+    help="Format of the output (default: credentials_process)",
+)
+@click.option("--profile", help="The AWS profile to use.")
+@_handle_error
+def queue_export_credentials(mode, output_format, **args):
+    """
+    Export queue credentials in a format compatible with AWS SDK credentials_process.
+    """
+    start_time = time.time()
+    is_success = True
+    error_type = None
+
+    # Get a temporary config object with the standard options handled
+    config = _apply_cli_options_to_config(required_options={"farm_id", "queue_id"}, **args)
+
+    queue_id: str = config_file.get_setting("defaults.queue_id", config=config)
+    farm_id: str = config_file.get_setting("defaults.farm_id", config=config)
+
+    try:
+        # Call the appropriate API based on mode
+        if mode.upper() == "USER":
+            response = api.assume_queue_role_for_user(
+                farmId=farm_id, queueId=queue_id, config=config
+            )
+        elif mode.upper() == "READ":
+            response = api.assume_queue_role_for_read(
+                farmId=farm_id, queueId=queue_id, config=config
+            )
+        else:
+            is_success = False
+            error_type = "InvalidMode"
+            raise DeadlineOperationError(f"Invalid mode: {mode}")
+
+        # Format the response according to the AWS SDK credentials_process format
+        if output_format.lower() == "credentials_process":
+            credentials = response["credentials"]
+            formatted_credentials = {
+                "Version": 1,
+                "AccessKeyId": credentials["accessKeyId"],
+                "SecretAccessKey": credentials["secretAccessKey"],
+                "SessionToken": credentials["sessionToken"],
+                "Expiration": credentials["expiration"].isoformat(),
+            }
+            click.echo(json.dumps(formatted_credentials))
```

Comment:
The output will be slightly nicer to look at if you add indent=1 or indent=2 in the dumps call.